specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json

```diff
@@ -513,7 +513,7 @@
             "modelAsString": true
           }
         },
-        "negationConditon": {
```

Comment:
It looks like this shouldn't be fixed: https://github.com/Azure/azure-rest-api-specs/pull/5341#discussion_r263686120
One of CI failures is [related to this](https://dev.azure.com/azure-sdk/public/_build/results?buildId=20119&view=logs&jobId=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&taskId=4f68196f-0a37-543a-e233-ac6814560e63&lineStart=2004&lineEnd=2004&colStart=1&colEnd=12)

+@azhao155 to confirm we still need to keep this typo to match server side. If no, we need to update examples then.

Reply:
reverted, will have mistyping saved

Reply:
Why not fix it if the service hasn't been released yet?